stability-tests/many-tips/main.go

```diff
@@ -271,13 +255,23 @@ func mineLoopUntilHavingOnlyOneTipInDAG(rpcAddress string, miningAddress util.Ad
 	if err != nil {
 		return errors.Wrapf(err, "Error in getCurrentTipsLength")
 	}
-	for numOfTips > 1 {
+	hasTimedOut := false
+	for numOfTips > 1 && !hasTimedOut {
 		time.Sleep(1 * time.Second)
 		numOfTips, err = getCurrentTipsLength(rpcClient)
 		if err != nil {
 			return errors.Wrapf(err, "Error in getCurrentTipsLength")
 		}
 	}
+	spawn("ChecksIfTimeIsUp", func() {
+		timer := time.NewTimer(20 * time.Minute)
+		<-timer.C
+		hasTimedOut = true
+	})
+
+	if hasTimedOut {
+		return errors.Errorf("Out of time - the graph still has more than one tip.")
```

Comment:
This and the check in line 203 seem to do the same thing. Please remove the latter.